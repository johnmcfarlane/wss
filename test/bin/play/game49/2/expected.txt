 38  4,12 - QADIS
 33 10,2  | QAID
 32  8,1  | QADI
 31 13,5  | QAID
 30 10,1  | QADI
 27  5,5  | DIPL
 27 10,1  | PAGOD
 27  8,12 - SAPO
 26  8,3  | QI
 25 11,6  | DOPA
 25 10,2  | GPAD
 25  5,6  | IPL
 25 13,6  | QADI
 25  8,12 - SAGO
 24 13,6  | PODIA
 24  8,12 - SAP
 23 11,7  | OPA
 22 10,12 - PA
 22 10,12 - PI
 22 10,12 - PO
 22 10,1  | PODIA
 22  8,12 - SAG
 21 10,2  | APOD
 21 10,2  | PAID
 20 10,2  | DIAG
 20  8,1  | DOPA
 20 10,1  | GAPO
 20 10,12 - GI
 20 10,12 - GO
 20 13,5  | GPAD
 20 11,4  | PAGOD
 19  5,10 | APOD
 19 11,6  | GAP
 19 10,2  | GOAD
 19 11,5  | GPAD
 19 10,2  | IGAD
 19 11,7  | ODA
 19 13,4  | PAGOD
 19 13,5  | PAID
 19 11,3  | PODIA
 19 13,4  | QADI
 18 10,7  - APHODI
 18 11,6  | DAP
 18 11,6  | DOP
 18 10,1  | DOPA
 18  8,1  | GADI
 18 13,6  | GADI
 18 12,10 - HAPI
 18 12,10 - HOPI
 17 11,5  | APOD
 17  4,4  | DAPICO
 17 11,5  | DIAG
 17  4,12 - GADIS
 17 13,5  | GOAD
 16  5,10 | ADP
 16  5,10 | AGIO
 16 13,5  | APOD
 16  4,8  | COGIDA
 16 11,6  | DAG
 16 10,1  | DAGO
 16 11,6  | DOG
 16  4,12 - DOPAS
 16 10,1  | GADI
 16 11,5  | GOAD
 16 12,10 - HAGI
 16 12,10 - HAP
 16 12,10 - HIP
 16 12,10 - HOGA
 16 12,10 - HOP
 16 11,5  | IGAD
 16 11,7  | OP
 16  8,2  | OPA
 16 11,8  | PA
 16 11,6  | PAD
 16  8,2  | PIA
 16 11,6  | POD
 16  8,2  | POI
 16  8,4  - PTG
 16  9,4  - TAIPO
 15 13,7  | ADO
 15 10,7  - APHID
 15  4,8  | COQ
 15  4,12 - DAGOS
 15 13,8  | DIA
 15 13,8  | DOA
 15  9,9  | DORIA
 15 11,6  | GAD
 15 11,4  | GAPO
 15 11,6  | GOD
 15 13,5  | IGAD
 15  5,10 | OPA
 15 13,3  | PODIA
 15 13,6  | QI
 15  6,12 - QIS
 15 10,10 | ZAP
 14 11,7  | AG
 14 10,1  | AGIO
 14  5,10 | AGO
 14  8,4  - ATIP
 14  8,4  - ATOP
 14 11,4  | DOPA
 14  9,9  | DORI
 14 13,4  | GAPO
 14  8,2  | GOA
 14  8,2  | GOI
 14 12,9  - GPAD
 14 10,7  - GPH
 14 12,10 - HAG
 14 12,10 - HOG
 14 10,1  | IAGO
 14  5,10 | OP
 14 10,2  | OPA
 14  8,3  | PA
 14 13,6  | PAD
 14  8,3  | PI
 14 10,2  | PIA
 14 13,5  | PIA
 14 10,3  | PIG
 14 13,6  | POD
 14 10,2  | POI
 14 13,5  | POI
 14 10,8  - QAID
 14 10,10 | ZAG
 13  5,10 | ADO
 13  5,10 | AG
 13 13,4  | AGIO
 13 13,5  | AGIO
 13  5,10 | AID
 13  4,12 - APIOS
 13  4,8  | COAID
 13  4,8  | COPIA
 13 11,4  | DAGO
 13 13,4  | DOPA
 13 13,6  | GAD
 13 13,6  | GID
 13 11,6  | GOA
 13 13,6  | GOD
 13 11,6  | OAD
 13  5,10 | OAD
 13  5,10 | ODA
 13  4,10 - PAX
 13  6,12 - PISA
 13  4,10 - PIX
 13  4,10 - POX
 13 10,3  | QI
 12 11,7  | AD
 12  5,10 | AD
 12 11,4  | AGIO
 12  4,12 - AGIOS
 12 10,2  | AGO
 12 10,7  - APH
 12 11,8  | DA
 12 10,3  | DAG
 12 13,4  | DAGO
 12 11,6  | DAO
 12  8,2  | DIA
 12 10,3  | DIG
 12  8,2  | DOA
 12 11,6  | DOA
 12 10,3  | DOG
 12 13,4  | GADI
 12  8,3  | GI
 12 10,2  | GIO
 12 13,5  | GIO
 12 10,2  | GOA
 12 13,5  | GOA
 12 10,2  | GOI
 12 13,5  | GOI
 12  4,10 - GOX
 12  8,4  - GTD
 12 12,10 - HAD
 12 12,10 - HID
 12 12,10 - HOD
 12 11,4  | IAGO
 12 10,7  - IPH
 12 11,7  | OD
 12  5,10 | OD
 12  8,2  | ODA
 12 11,5  | OPA
 12 10,3  | PAD
 12 11,7  - PHI
 12 11,7  - PHO
 12 11,5  | PIA
 12 10,3  | POD
 12  8,4  - PTA
 12  9,4  - TAIG
 12  9,4  - TOAG
 11  4,12 - ADIOS
 11 11,5  | AGO
 11  5,10 | AI
 11 13,6  | AID
 11  9,5  - APAID
 11  9,5  - APODIA
 11 11,9  - DGAG
 11  6,12 - DISA
 11  9,9  | DOR
 11  6,12 - DOSA
 11 10,3  | GAD
 11 12,9  - GAPO
 11 10,3  | GID
 11 11,5  | GIO
 11 13,6  | GIO
 11 11,5  | GOA
 11 12,9  - GOAD
 11 10,3  | GOD
 11 12,7  - HAPI
 11 12,7  - HOPI
 11 13,4  | IAGO
 11 13,6  | OAD
 11  5,10 | OI
 11 13,5  | OPA
 11 11,6  | PA
 11 11,6  | PO
 11 11,8  - QI
 11 11,11 | TOPI
 11  6,9  - aMID
 10  4,7  | ACOP
 10 13,7  | AD
 10  9,5  - ADAGIO
 10 10,2  | ADO
 10 11,5  | ADO
 10  9,5  - ADPAO
 10 10,4  | AG
 10 13,5  | AGO
 10 10,8  - APIOID
 10  5,12 - APIS
 10  9,5  - APODA
 10  4,8  | CAPO
 10  8,3  | DA
 10 10,2  | DAO
 10 11,5  | DAO
 10 13,5  | DAO
 10 13,6  | DAO
 10  8,3  | DI
 10 13,8  | DI
 10 10,2  | DIA
 10 11,5  | DIA
 10 13,5  | DIA
 10 13,8  | DO
 10 10,2  | DOA
 10 11,5  | DOA
 10 13,5  | DOA
 10  7,11 | DOPA
 10 11,7  - GHI
 10 12,9  - GIGA
 10 11,6  | GO
 10 12,7  - HAGI
 10 12,10 - HAO
 10 12,10 - HIA
 10 12,7  - HOGA
 10 12,10 - HOI
 10 13,7  | ID
 10 13,9  | IO
 10 13,7  | OD
 10 10,2  | ODA
 10 11,5  | ODA
 10 13,9  | OI
 10 10,7  - PAHI
 10 10,7  - PAHO
 10  5,12 - PAIS
 10 10,3  | PIA
 10  5,12 - PIAS
 10  4,6  | PICA
 10  4,6  | PICO
 10 10,7  - POHA
 10 10,3  | POI
 10  5,12 - POIS
 10 11,11 | TAP
 10 11,11 | TIP
 10 11,11 | TOP
  9 13,5  | ADO
  9 11,9  - AGOG
  9  5,12 - AGOS
  9 10,3  | AID
  9  9,9  | AOR
  9  5,10 - AX
  9  4,8  | CAP
  9  4,8  | COP
  9 11,6  | DA
  9  7,11 | DAGO
  9  7,11 | DAP
  9 11,7  - DHAI
  9  7,11 | DIAG
  9  7,11 | DIP
  9 11,6  | DO
  9  7,11 | DOP
  9 12,9  - GADI
  9 10,3  | GOA
  9  5,12 - GOAS
  9 10,3  | GOI
  9  5,12 - GOIS
  9  5,5  - GOV
  9 12,9  - GPD
  9 11,7  - IHP
  9 10,3  | OAD
  9 13,5  | ODA
  9  5,10 - OX
  9 13,6  | PA
  9  4,6  | PAC
  9 10,7  - PAH
  9  6,12 - PAS
  9 13,6  | PI
  9  4,6  | PIC
  9  6,12 - PIS
  9 13,6  | PO
  9 10,7  - POH
  9  6,12 - POS
  9 11,11 | TAIPO
  9  9,4  - TOAD
  9 11,11 | TOGA
  9  9,4  - TOPI
  9  6,9  - aMIA
  8  4,7  | ACID
  8 10,4  | AD
  8  5,12 - ADOS
  8  5,5  - ADV
  8  8,3  | AI
  8  9,5  - APOD
  8  4,8  | CADI
  8  4,8  | CAG
  8  4,8  | CAID
  8  4,8  | CIG
  8  4,8  | CODA
  8  4,8  | COG
  8  7,11 | DADO
  8  7,11 | DAG
  8 10,7  - DAHI
  8  5,12 - DAIS
  8 11,7  - DHA
  8 10,3  | DIA
  8  5,12 - DIAS
  8  7,11 | DIDO
  8  7,11 | DIG
  8  4,5  | DIOC
  8  5,5  - DIV
  8 10,3  | DOA
  8  7,11 | DOG
  8 12,9  - GAP
  8  6,12 - GAS
  8 13,6  | GI
  8 11,8  - GIP
  8 12,9  - GIP
  8  6,12 - GIS
  8 13,6  | GO
  8 12,7  - HAP
  8 12,7  - HIP
  8 12,7  - HOP
  8  4,7  | ICOD
  8 10,4  | ID
  8 11,9  - IGAD
  8 11,6  | IO
  8 13,7  | IO
  8 10,8  - IPID
  8 12,8  - IPID
  8  8,4  - ITD
  8 10,4  | OD
  8  4,5  | ODIC
  8  8,3  | OI
  8 13,8  | OI
  8  9,10 | ORIG
  8 10,8  - PAID
  8 11,8  - PIG
  8 11,11 | TAG
  8 11,11 | TIG
  8 11,11 | TOG
  8  9,4  - TOGA
  7  9,5  - ADP
  7  9,5  - AGAD
  7 11,9  - AGIO
  7 10,8  - APII
  7  4,8  | CAD
  7  4,8  | CIAO
  7  4,8  | CID
  7  4,8  | COD
  7 13,6  | DA
  7  7,11 | DAD
  7 10,7  - DAH
  7  6,12 - DAS
  7  4,7  | DCA
  7 13,6  | DI
  7 11,8  - DIAG
  7  7,11 | DID
  7 11,8  - DIP
  7  6,12 - DIS
  7 13,6  | DO
  7  4,6  | DOC
  7  7,11 | DOD
  7 10,7  - DOH
  7  6,12 - DOS
  7 12,9  - GAG
  7 12,9  - GIG
  7 12,9  - GOG
  7 12,7  - HAG
  7 12,7  - HOG
  7 12,8  - IGAD
  7 11,7  - OHIA
  7 10,3  | PA
  7 10,3  | PI
  7 10,3  | PO
  7 11,11 | TAIG
  7 11,11 | TOAG
  7  9,4  - TODA
  7 11,11 | TODA
  7 11,11 | TPI
  6  9,5  - AGIO
  6 10,8  - AGIO
  6 11,9  - AGO
  6 11,7  - AHI
  6 11,7  - AHO
  6 10,4  | AI
  6 13,6  | AI
  6  6,12 - AIS
  6  9,5  - APA
  6  8,4  - ATI
  6  4,8  | CAI
  6  4,8  | CIA
  6  7,11 | DAO
  6  7,11 | DIA
  6 11,8  - DIG
  6  7,11 | DOA
  6 12,9  - GAD
  6 10,3  | GI
  6 11,8  - GID
  6 12,9  - GID
  6 10,3  | GO
  6 12,9  - GOD
  6 12,7  - HAD
  6 12,7  - HID
  6 12,7  - HOD
  6 12,8  - IAGO
  6 13,6  | IO
  6  8,4  - ITA
  6  4,7  | OCA
  6 10,4  | OI
  6 13,6  | OI
  6 11,8  - OIDIA
  6 11,8  - PIA
  6 10,8  - POI
  6 11,11 | TAD
  6  9,4  - TAP
  6  9,4  - TIAO
  6  9,4  - TIP
  6 11,11 | TOAD
  6 11,11 | TOD
  6  9,4  - TOP
  6  9,4  - TPI
  5  9,5  - ADAI
  5 11,9  - AG
  5  9,5  - AGA
  5  9,5  - AGO
  5 11,7  - AH
  5  7,12 - AS
  5 10,3  | DA
  5  7,11 | DA
  5 10,3  | DI
  5  7,11 | DI
  5 10,3  | DO
  5  7,11 | DO
  5 11,8  - GIO
  5 12,9  - GIO
  5 12,9  - GOA
  5 10,8  - GOI
  5 12,9  - GOI
  5 12,7  - HAO
  5 12,7  - HIA
  5 12,7  - HOI
  5  7,12 - IS
  5 11,7  - OH
  5  7,12 - OS
  5 11,8  - PI
  5  9,4  - TAG
  5 11,11 | TIAO
  5  9,4  - TIG
  5  9,4  - TOG
  4  9,5  - ADO
  4  9,5  - AG
  4 10,3  | AI
  4  9,5  - AID
  4 11,8  - AID
  4  8,4  - AT
  4 11,8  - DIA
  4 11,8  - GI
  4 12,9  - GI
  4 12,9  - GO
  4 12,7  - HA
  4 12,10 - HA
  4 12,7  - HI
  4 12,10 - HI
  4 12,7  - HO
  4 12,10 - HO
  4 10,3  | IO
  4  8,4  - IT
  4 10,3  | OI
  4  9,4  - TAD
  4 11,11 | TAI
  4 11,11 | TAO
  4 11,11 | TOA
  4  9,4  - TOD
  4 11,11 | TOI
  3  9,5  - AD
  3 11,8  - DI
  3 12,8  - ID
  3  9,4  - TAI
  3  9,4  - TAO
  3  9,4  - TOA
  3  9,4  - TOI
  2  9,5  - AA
  2  9,5  - AI
  2 11,8  - AI
  2 12,8  - IO
  2 11,8  - OI
  2  9,4  - TA
  2 11,11 | TA
  2  9,4  - TI
  2 11,11 | TI
  2  9,4  - TO
  2 11,11 | TO
