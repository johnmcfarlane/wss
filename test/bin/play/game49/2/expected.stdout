 38  4,12 - QADIS
 33 10,2  | QAID
 32  8,1  | QADI
 31 13,5  | QAID
 30 10,1  | QADI
 27 10,1  | PAGOD
 26  8,3  | QI
 25 11,6  | DOPA
 25 13,6  | QADI
 25  8,12 - SAGO
 24 13,6  | PODIA
 24  8,12 - SAP
 22 10,12 - PA
 22 10,12 - PI
 22 10,12 - PO
 22 10,1  | PODIA
 22  8,12 - SAG
 21 10,2  | APOD
 21 10,2  | PAID
 20  8,1  | DOPA
 20 10,12 - GI
 20 10,12 - GO
 20  8,1  | PADI
 20  4,12 - PADIS
 20 11,4  | PAGOD
 19 13,7  | APO
 19  5,10 | APOD
 19 11,6  | GAP
 19 10,2  | GOAD
 19 11,7  | ODA
 19 13,6  | PADI
 19 13,4  | PAGOD
 19 13,5  | PAID
 19 13,8  | PIA
 19 11,3  | PODIA
 19 13,8  | POI
 19 13,4  | QADI
 18 11,6  | DAP
 18 10,1  | DOPA
 18  8,1  | GADI
 18 13,6  | GADI
 18 10,1  | PADI
 17 11,5  | APOD
 17  4,12 - GADIS
 17 13,6  | GAP
 17 13,6  | GIP
 17 13,5  | GOAD
 16  5,10 | AGIO
 16 13,5  | APOD
 16 11,6  | DAG
 16 10,1  | DAGO
 16  1,9  - DAGO
 16 13,6  | DAP
 16 13,6  | DIP
 16 11,6  | DOG
 16  4,12 - DOPAS
 16 10,1  | GADI
 16 11,5  | GOAD
 16 12,10 - HAP
 16 12,10 - HIP
 16 12,10 - HOP
 16 11,7  | OP
 16 11,8  | PA
 16 13,9  | PA
 16 11,6  | PAD
 16 13,9  | PI
 16  8,2  | PIA
 16 13,9  | PO
 16 11,6  | POD
 16  8,2  | POI
 15 13,7  | ADO
 15 10,7  - APHID
 15  5,10 | APO
 15  4,8  | COQ
 15  4,12 - DAGOS
 15 11,6  | GAD
 15 11,6  | GOD
 15 13,3  | PODIA
 15 13,6  | QI
 15  6,12 - QIS
 15 10,10 | ZAP
 14 11,7  | AG
 14 10,1  | AGIO
 14  5,10 | AGO
 14 10,2  | APO
 14  8,4  - ATOP
 14 11,4  | DOPA
 14  8,2  | GOA
 14 12,10 - HAG
 14 12,10 - HOG
 14 13,7  | IP
 14 13,7  | OP
 14  5,10 | OP
 14  8,3  | PA
 14 13,6  | PAD
 14  8,3  | PI
 14 13,8  | PI
 14 10,2  | PIA
 14 13,5  | PIA
 14 10,3  | PIG
 14 13,8  | PO
 14 13,6  | POD
 14 10,2  | POI
 14 13,5  | POI
 14 10,8  - QAID
 14 10,10 | ZAG
 13  5,10 | ADO
 13  5,10 | AG
 13 13,4  | AGIO
 13  1,9  - AGIO
 13  5,10 | AID
 13 11,4  | DAGO
 13 13,4  | DOPA
 13 13,6  | GAD
 13 13,6  | GID
 13 11,6  | GOA
 13 13,6  | GOD
 13 11,6  | OAD
 13  5,10 | OAD
 13  5,10 | ODA
 13 13,4  | PADI
 13  4,10 - PAX
 13  4,10 - PIX
 13  4,10 - POX
 13 10,3  | QI
 12 11,7  | AD
 12  5,10 | AD
 12 11,4  | AGIO
 12  4,12 - AGIOS
 12 10,2  | AGO
 12 11,5  | APO
 12 11,8  | DA
 12 10,3  | DAG
 12 13,4  | DAGO
 12 10,3  | DIG
 12 10,3  | DOG
 12 13,4  | GADI
 12  8,3  | GI
 12 10,2  | GOA
 12 13,5  | GOA
 12  4,10 - GOX
 12 12,10 - HAD
 12 12,10 - HID
 12 12,10 - HOD
 12 11,7  | OD
 12  5,10 | OD
 12  8,2  | ODA
 12 10,3  | PAD
 12 11,7  - PHI
 12 11,5  | PIA
 12 10,3  | POD
 11  4,12 - ADIOS
 11 11,5  | AGO
 11  5,10 | AI
 11 13,6  | AID
 11 13,5  | APO
 11  2,9  - APO
 11  9,9  | DOR
 11 10,3  | GAD
 11 10,3  | GID
 11 11,5  | GOA
 11 12,9  - GOAD
 11 10,3  | GOD
 11 13,6  | OAD
 11  5,10 | OI
 11 11,6  | PA
 11 11,6  | PO
 11 11,8  - QI
 11 11,11 | TOPI
 11  6,9  - aMID
 10 13,7  | AD
 10  9,5  - ADAGIO
 10 10,2  | ADO
 10 11,5  | ADO
 10 10,4  | AG
 10 13,5  | AGO
 10  2,9  - AGO
 10  4,10 - AIX
 10  5,12 - APOS
 10  4,8  | CAPO
 10  8,3  | DA
 10  8,3  | DI
 10 13,8  | DI
 10 13,8  | DO
 10  7,11 | DOPA
 10 11,7  - GHI
 10 12,9  - GIGA
 10 11,6  | GO
 10 12,10 - HAO
 10 12,10 - HOI
 10 13,7  | ID
 10 13,7  | OD
 10 10,2  | ODA
 10 11,5  | ODA
 10 13,9  | OI
 10 10,3  | PIA
 10  5,12 - PIAS
 10  4,6  | PICA
 10  3,9  - PO
 10 10,3  | POI
 10  5,12 - POIS
 10 11,11 | TAP
 10 11,11 | TIP
 10 11,11 | TOP
  9 13,5  | ADO
  9  2,9  - ADO
  9 11,9  - AGOG
  9 10,3  | AID
  9  5,10 - AX
  9  4,8  | CAP
  9  4,8  | COP
  9  4,8  | CPA
  9 11,6  | DA
  9  7,11 | DAGO
  9  7,11 | DAP
  9  7,11 | DIP
  9 11,6  | DO
  9 12,9  - GADI
  9  3,9  - GO
  9 10,3  | GOA
  9  5,12 - GOAS
  9 10,3  | OAD
  9 13,5  | ODA
  9  5,10 - OX
  9 13,6  | PA
  9  4,6  | PAC
  9 10,7  - PAH
  9  6,12 - PAS
  9 13,6  | PI
  9  4,6  | PIC
  9  6,12 - PIS
  9 13,6  | PO
  9 10,7  - POH
  9  6,12 - POS
  9  9,4  - TOAD
  9 11,11 | TOGA
  9  9,4  - TOPI
  8  4,7  | ACID
  8 10,4  | AD
  8  5,12 - ADOS
  8  8,3  | AI
  8  9,5  - APOD
  8  4,8  | CADI
  8  4,8  | CAID
  8  4,8  | CIG
  8  4,8  | CODA
  8  4,8  | COG
  8  7,11 | DADO
  8  7,11 | DAG
  8  5,12 - DAIS
  8  7,11 | DIDO
  8  7,11 | DIG
  8  3,9  - DO
  8  7,11 | DOG
  8 12,9  - GAP
  8  6,12 - GAS
  8 13,6  | GI
  8 11,8  - GIP
  8 12,9  - GIP
  8 13,6  | GO
  8 12,7  - HAP
  8 12,7  - HIP
  8 12,7  - HOP
  8 10,4  | ID
  8 11,9  - IGG
  8 10,4  | OD
  8  5,12 - ODAS
  8  4,5  | ODIC
  8  8,3  | OI
  8 10,8  - PAID
  8 11,8  - PIG
  8 11,11 | TAG
  8 11,11 | TOG
  8  9,4  - TOGA
  7 11,9  - AGIO
  7  4,8  | CAD
  7  4,8  | CIAO
  7  4,8  | COD
  7 13,6  | DA
  7  7,11 | DAD
  7 10,7  - DAH
  7 13,6  | DI
  7  7,11 | DID
  7 11,8  - DIP
  7  6,12 - DIS
  7 13,6  | DO
  7  4,6  | DOC
  7 10,7  - DOH
  7  6,12 - DOS
  7 12,9  - GAG
  7 12,9  - GIG
  7 12,9  - GP
  7 12,7  - HAG
  7 12,7  - HOG
  7 11,7  - OHIA
  7  9,10 | ORAD
  7 10,3  | PA
  7 10,3  | PI
  7 10,3  | PO
  6  9,5  - AGIO
  6 10,8  - AGIO
  6 11,9  - AGO
  6 11,7  - AHI
  6 10,4  | AI
  6 13,6  | AI
  6  6,12 - AIS
  6  9,5  - APO
  6 11,8  - DIG
  6 12,9  - GAD
  6 10,3  | GI
  6 11,8  - GID
  6 12,9  - GID
  6 10,3  | GO
  6 12,9  - GOD
  6 12,7  - HAD
  6 12,7  - HID
  6 12,7  - HOD
  6  4,7  | OCA
  6 10,4  | OI
  6 13,6  | OI
  6 11,8  - OIDIA
  6 11,8  - PIA
  6 10,8  - POI
  6 11,11 | TAD
  6  9,4  - TAP
  6  9,4  - TIP
  6 11,11 | TOAD
  6 11,11 | TOD
  6  9,4  - TOP
  5 11,9  - AG
  5  9,5  - AGA
  5  9,5  - AGO
  5 11,7  - AH
  5  7,12 - AS
  5  4,8  | CO
  5 10,3  | DA
  5  7,11 | DA
  5 10,3  | DI
  5  7,11 | DI
  5 10,3  | DO
  5  7,11 | DO
  5 12,9  - GOA
  5 12,7  - HAO
  5 12,7  - HOI
  5 12,8  - IP
  5  7,12 - IS
  5 11,7  - OH
  5  9,10 | ORA
  5  7,12 - OS
  5 11,8  - PI
  5  8,12 - SA
  5  9,4  - TAG
  5  9,4  - TOG
  4  9,5  - ADO
  4  9,5  - AG
  4 10,3  | AI
  4  9,5  - AID
  4 11,8  - AID
  4  8,4  - AT
  4 11,8  - GI
  4 12,9  - GI
  4 12,9  - GO
  4 12,7  - HA
  4 12,10 - HA
  4 12,7  - HI
  4 12,10 - HI
  4 12,7  - HO
  4 12,10 - HO
  4  8,4  - IT
  4 10,3  | OI
  4  9,4  - TAD
  4 11,11 | TAO
  4  9,4  - TOD
  3  9,5  - AD
  3 11,8  - DI
  3 12,8  - ID
  3  9,4  - TAO
  2  9,5  - AA
  2  9,5  - AI
  2 11,8  - AI
  2 11,8  - OI
  2  9,4  - TA
  2 11,11 | TA
  2  9,4  - TI
  2 11,11 | TI
  2  9,4  - TO
  2 11,11 | TO
